optuna/visualization/matplotlib/_contour.py

```diff
@@ -51,11 +50,6 @@ def plot_contour(
     .. seealso::
         Please refer to :func:`optuna.visualization.plot_contour` for an example.
 
-    Warnings:
```

Comment:
I think it would be user-friendly to leave this documentation because there is still difference between the plotly backend and this implementation. What do you think?

Reply:
I think they are similar, and no need to show the warning. But it is not the main task of this PR. Let me revert the removal.